### ISSUE TITLE
Website credits

### ISSUE THREAD

**Issue by cchristopherhowardd** [aarome/aarome#336]
How about giving credit to the designers and developers in the footer? Right under the copyright line it could say:

Designed by Open and developed by Jason Loeffler/Minor Science/your credit.

Links go to http://notclosed.com/ for "Open" and whatever website you use (LinkedIn?).

**Comment by jasloe** [comment]
Yep, it's on the list. Would prefer to do this when we've squashed all (or at least most) of my bugs!

**Comment by cchristopherhowardd** [comment]
It just seems like an easy thing to do, that's all.
